src/ebook-speaker.c

```diff
@@ -2926,8 +2926,8 @@ int main (int argc, char *argv[])
    noecho ();
    misc.player_pid = -2;
    sprintf (misc.scan_resolution, "400");
-   snprintf (misc.copyright, MAX_STR - 1, "%s %s - (C)2021 J. Lemmens",
-             gettext ("eBook-speaker - Version"), PACKAGE_VERSION);
+   snprintf (misc.copyright, MAX_STR - 1, "%s                        ",
+             gettext ("eBook-speaker          "));
```

Comment:
You should use `git rebase -i origin/master` to drop the other commit from this branch